src/promptflow/promptflow/_utils/load_data.py

```diff
@@ -59,12 +59,13 @@ def _bfs_dir(dir_path: List[str]) -> Tuple[List[str], List[str]]:
     """BFS traverse directory with depth 1, returns files and directories"""
     files, dirs = [], []
     for path in dir_path:
-        for filename in os.listdir(path):
-            file = Path(path, filename).resolve()
-            if file.is_file():
-                files.append(str(file))
-            else:
-                dirs.append(str(file))
+        if os.path.exists(path):
+            for filename in os.listdir(path):  # Path does not exist and will throw FileNotFoundError
```

Comment:
I think we should try catch and throw this as UserError?
If we suppress this message we lose some information about the folder?

Reply:
Based on the context message, the purpose of this function is to search for valid files and directories, so I prefer to add a check to see if the directory is valid instead of throwing an error.